check_warnings.sh

```diff
@@ -6,27 +6,18 @@
 # Desc: Check for warnings during build process of repo $CI_SOURCE_PATH
 
 packages_with_warnings() {
-   for pkg in $(colcon info | grep 'name: ' | sed -e "s/.*name: //g" 2> /dev/null) ; do
-      # Warnings manifest themselves log files in catkin tools' logs folder
-      log_file=$(find $ROS_WS/log/latest_build/$pkg -name "stderr.log" 2> /dev/null)
-      # Check if the stderr.log file is not empty and add it to the list of warnings if it is
-      # Print result
-      if [ -s ${log_file} ]; then
-         echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-         echo "  Test results for: $pkg"
-         echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-         echo -e "- $(colorize YELLOW $(colorize THIN $pkg)): $log_file"
-         echo ""
-         cat $log_file
-         echo ""
-      fi
+   SOURCE_PKGS=($(colcon list --topological-order --names-only --base-paths $CI_SOURCE_PATH 2> /dev/null))
+   for pkg in ${SOURCE_PKGS[@]} ; do
+      # Warnings manifest themselves with a non-empty stderr.log file in colcon's log folder
+      log_file=$ROS_WS/log/latest_build/$pkg/stderr.log
+      test -s "$log_file" && echo -e "- $(colorize YELLOW $(colorize THIN $pkg))"
```

Comment:
I liked that this presented the user with the warnings that caused the failure. Why revert it?

Reply:
Because the warnings are already printed in the build section. The output here was only intended as a summary. In Travis we need to carefully save lines of output, because only 10.000 lines are allowed...

Reply:
Ok. I see that ` --event-handlers desktop_notification- status-` accomplishes that. Personally, I would prefer if we didn't output the warnings to console during build an instead printed them at the end as I had it before.but I see that that is a matter of opinion.

Reply:
Note: It's not handled by `desktop_notification- status-`, but the default `log+`. So, warnings and errors were printed before as well.